plugins/automox/plugin.spec.yaml

```
@@ -0,0 +1,988 @@
+      required: false
+    policy_type_name:
+      title: Policy Type Name
+      description: The type of policy
+      type: string
+      required: false
+    configuration:
+      title: Policy Configuration
+      description: The configuration of the policy
+      type: object
+      required: false
+    server_groups:
+      title: Device Groups
+      description: List of device groups assigned policy
+      type: "[]integer"
+      required: false
+  group:
+    id:
+      title: Group ID
+      description: The group identifier
+      type: integer
+      required: true
+    organization_id:
+      title: Organization ID
+      description: The organization identifier of the group
+      type: integer
+      required: true
+    name:
+      title: Group Name
+      description: The name of the group
+      type: string
+      required: false
+    refresh_interval:
+      title: Refresh Interval
+      description: Frequency of device refreshes in minutes
+      type: integer
+      required: false
+    parent_server_group_id:
+      title: Parent Server Group ID
+      description: The identifier of the parent group
+      type: integer
+      required: false
+    ui_color:
+      title: Color
+      description: Automox console highlight color for the group
+      type: string
+      required: false
+    notes:
+      title: Notes
+      description: Notes defined for the group
+      type: string
+      required: false
+    enable_os_auto_update:
+      title: Enable OS Auto Update
+      description: Enable operating system auto update
+      type: boolean
+      required: false
+    server_count:
+      title: Device Count
+      description: Number of devices assigned to group
+      type: integer
+      required: false
+    policies:
+      title: Policies
+      description: List of policies assigned to group
+      type: "[]integer"
+      required: false
+  event:
+    id:
+      title: Event ID
+      description: Identifier of event
+      type: integer
+      required: true
+    name:
+      title: Event name
+      description: Name of event
+      type: string
+      required: true
+    user_id:
+      title: User ID
+      description: Identifier of user
+      type: integer
+      required: false
+    server_id:
+      title: Device ID
+      description: Identifier of device
+      type: integer
+      required: false
+    organization_id:
+      title: Organization ID
+      description: Identifier of organization
+      type: integer
+      required: false
+    policy_id:
+      title: Policy ID
+      description: Identifier of policy
+      type: integer
+      required: false
+    data:
+      title: Event Data
+      description: Data included with event
+      type: object
+      required: false
+    server_name:
+      title: Device Name
+      description: Name of device
+      type: string
+      required: false
+    policy_name:
+      title: Policy Name
+      description: Name of policydentifier of user
```

```suggestion
      description: Name of policy
```
I think? Unfamiliar with "policydentifier"

---

@ax-zyoutz is this a typo?